### ISSUE TITLE
New WOComponent templates should go to the components folder

### ISSUE THREAD

**Issue by hugith** [wocommunity/wolips#152]
When a new WOComponent is created in maven projects the component template (.wo) is saved to the root of the project by default. It should default to src/main/components.

**Comment by nullterminated** [comment]
This sounds a lot easier than it is. For one, the components dir could be configured differently in a maven project, so it would need to read the maven model to really know where to put them. Secondly, it's not uncommon for components to be localized and nested in a English.lproj or NonLocalized.lproj directory.

But I agree, it is totally annoying that it ends up in the project root if you aren't paying close attention when you create components.

**Comment by hugithordarson** [comment]
Yeah, I see the problem. Might cover most cases if it tries to default to "Components" in the project root, and if that doesn't exist, then check for src/main/components? I'd think that covers most projects (don't think custom maven project structures are common (other than Fluffy Bunny)). And for localized project structures, it's at least easier to navigate to the localized subfolders if the component creation window starts out with the component root folder selected.

In any case, not a big issue on my behalf, merely an annoyance that I've gotten used to :).
